Add a FeatureToggle bootstrapper. (#131)
Unleash/unleash-client-java: README.md

* When `unleash-repo.json` does not exists
* When the named feature toggle does not exist in `unleash-repo.json`

## Bootstrapping

> Unleash supports bootstrapping from a local file where Unleash does not have write access.

Author: ```suggestion
Unleash supports bootstrapping from a local file without need for write access.
```

Useful for applications deployed to ephemeral containers (docker) or more locked down file systems where Unleash's need to write back to the backup file is not desired.
Configure using
* `UNLEASH_BOOTSTRAP_FILE` - path to file in toggle json format as defined by the Unleash servers `/api/client/features` endpoint

> * `UNLEASH_BOOTSRAP_FILE_CHECKSUM`

Author: ```suggestion
* `UNLEASH_BOOTSTRAP_FILE_CHECKSUM`
```
